src/wombats/handlers/auth.clj

```diff
@@ -149,9 +150,11 @@
                  ;; Check if the user has a vaild access key
                  (if (:user/access-key updated-user)
                    (redirect-home context referer user-access-token)
-                   (failed-redirect))))
-             (failed-redirect)))
-         (failed-redirect))))))
+                   (failed-redirect "Wombats is in Alpha. You'll need to request an early access token to get started now."))))
```

Comment:
"Oops! What's the magic word? Please request an access key from an administrator."